scripts/src/models/translations.ts

```diff
@@ -18,9 +20,15 @@ export class OfferTranslation extends BaseEntity {
 
 	public static async getTranslations(criteria: GetTranslationsCriteria = {}): Promise<OfferTranslation[]> {
 		//  todo add cache
+		const cacheKey = `${criteria.languages}:${criteria.offerId}:${criteria.paths}`;
```

Comment:
should be sorted(criteria.languages)

Reply:
The language order has meaning, language priority.

Reply:
paths can be sorted, but maybe let's deploy it this way, so it'll hit the DB once or twice more instead of fetching from cache